mate-panel/drawer.c

```diff
@@ -636,7 +639,7 @@ panel_drawer_create (PanelToplevel *toplevel,
 char *
 panel_drawer_create_with_id (const char    *toplevel_id,
                              int            position,
-                             const char    *custom_icon,
+                             GIcon         *custom_icon,
```

Comment:
Looks like `panel_drawer_create` above should be modified in the same way.

```C
drawer.c: In function ‘panel_drawer_create’:
drawer.c:632:31: warning: passing argument 2 of ‘panel_drawer_prepare’ from incompatible pointer type [-Wincompatible-pointer-types]
     panel_drawer_prepare (id, custom_icon, use_custom_icon, tooltip, NULL);
                               ^~~~~~~~~~~
drawer.c:570:1: note: expected ‘GIcon * {aka struct _GIcon *}’ but argument is of type ‘const char *’
 panel_drawer_prepare (const char  *drawer_id,
 ^~~~~~~~~~~~~~~~~~~~
```

Reply:
Thank you for review, It needs to be modified here.